src/main/java/com/blogify/blogapi/service/event/CategoryService.java

```diff
@@ -0,0 +1,17 @@
+package com.blogify.blogapi.service.event;
+
+import com.blogify.blogapi.repository.CategoryRepository;
+import com.blogify.blogapi.repository.model.Category;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@AllArgsConstructor
+public class CategoryService {
+  private final CategoryRepository categoryRepository;
+
+  public List<Category> getAll() {
+    return categoryRepository.findAll();
+  }
+}
```

Comment:
why in event directory? just put it directly in service

Reply:
that's my mistake, thank you for the remark